src/ghdcbot/adapters/github/rest.py

```
@@ -0,0 +1,979 @@
+                                        repo=repo,
+                                        created_at=reverted_merged_at,  # Use original merge time
+                                        payload={
+                                            "pr_number": reverted_pr_number,
+                                            "reverted_by_pr": pr["number"],
+                                            "reverted_at": pr.get("created_at"),
+                                        },
+                                    )
+                                )
+                    except Exception:  # noqa: BLE001
+                        # Network errors, etc. - skip revert detection
+                        pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)